src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,77 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+
+  margin-top: 8px;
+  margin-bottom: 8px;
+  column-gap: 17px;
+
+  &-amount {
+    display: flex;
+
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 16px;
+    text-align: right;
+
+    color: $main-text-color;
+  }
```

Comment:
You have `display: flex;` and `align-items: flex-end;` in `.card__review` and then again in `.card__review-amount`. This seems redundant. Please review this and remove unnecessary rules.